### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by Shab00** [Homebrew/discussions#5247] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.15
ORIGIN: https://github.com/Homebrew/brew
HEAD: 92a4311868322188478d7a90511ec0e8e6b0d7df
Last commit: 6 days ago
Core tap JSON: 30 Mar 12:19 UTC
Core cask tap JSON: 30 Mar 12:19 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.gWGo4jX6UU/org.xquartz:0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 14.0.3 build 1403
Git: 2.44.0 => /usr/local/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 13.6.6-x86_64
CLT: 14.3.1.0.1.1683849156
Xcode: 15.2
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Settings.

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 15.1.
```

### Description of issue

The brew install is seemingly broken. The protection on the plist is apparently incorrect. 

When i run brew start httpd i get:
Error: Permission denied @ rb_sysopen - /Users/bashaar/Library/LaunchAgents/homebrew.mxcl.httpd.plis

Is there anyway to solve this, as im trying to run apache server with mod-wsgi? or if any body can point me in the right direction?

Many thanks

**Comment by SMillerDev** [comment]
What does `ls -al /Users/bashaar/Library/LaunchAgents/homebrew.mxcl.httpd.plist` say?

**Comment by Shab00** [comment]
It says No such file or directory

**Reply by SMillerDev** [reply]
How about `ls -al /Users/bashaar/Library/LaunchAgents/`?

**Comment by Shab00** [comment]
Hi,

this is the ouput:

total 0
drwxr-xr-x   2 root     staff    64  1 Mar  2021 .
drwx------@ 96 bashaar  staff  3072 17 Mar 09:01 ..

thanks

**Reply by SMillerDev** [reply, marked as answer]
That's your problem. That should be owned by you, not root. You can run this to resolve it:
```
sudo chown -R bashaar /Users/bashaar/Library/LaunchAgents/
```

**Reply by Shab00** [reply]
So everything is running however, brew services shows an error:

httpd error  256 bashaar ~/Library/LaunchAgents/homebrew.mxcl.httpd.plist
php   none

and error_log shows:

[Wed Mar 27 08:35:12.526331 2024] [core:notice] [pid 67898] AH00094: Command line: '/usr/local/opt/httpd/bin/httpd -D FOREGROUND'
[Wed Mar 27 09:35:11.817494 2024] [mpm_prefork:notice] [pid 67898] AH00169: caught SIGTERM, shutting down
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 192.168.0.7. Set the 'ServerName' directive globally to suppress this message
[Wed Mar 27 09:35:13.069729 2024] [mpm_prefork:notice] [pid 92722] AH00163: Apache/2.4.58 (Unix) configured -- resuming normal operations
[Wed Mar 27 09:35:13.070487 2024] [core:notice] [pid 92722] AH00094: Command line: '/usr/local/opt/httpd/bin/httpd -D FOREGROUND'
[Thu Mar 28 06:59:17.523035 2024] [mpm_prefork:notice] [pid 92722] AH00169: caught SIGTERM, shutting down
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 192.168.0.7. Set the 'ServerName' directive globally to suppress this message
[Thu Mar 28 06:59:18.770334 2024] [mpm_prefork:notice] [pid 19772] AH00163: Apache/2.4.58 (Unix) configured -- resuming normal operations
[Thu Mar 28 06:59:18.771428 2024] [core:notice] [pid 19772] AH00094: Command line: '/usr/local/opt/httpd/bin/httpd -D FOREGROUND'
[Thu Mar 28 07:11:42.807512 2024] [mpm_prefork:notice] [pid 19772] AH00169: caught SIGTERM, shutting down

Im on mac intel

Would it be a case of reinstalling brew and packages?

thanks

**Comment by Shab00** [comment]
Thank you so much, was driving mad at one point. Yes Its working now!
